pkg/constants/constants.go

```diff
@@ -0,0 +1,6 @@
+package constants
+
+const (
+	//DefaultKubeVersion is the default kube version
+	DefaultKubeVersion = "1.19.0"
```

Comment:
IMHO, we should source this from Talos constants. That way we can default to the version that Talos was built against.